CFC_DataCollector/trip_analysis/kmedoid.py

```diff
@@ -0,0 +1,122 @@
+import random
+import sys
+
+#code based on K_medoid_2.py in CFC_WebApp/main
```

Comment:
Also, can you add a unit test to ensure that it continues working as expected even after you leave?